x-pack/plugins/triggers_actions_ui/public/application/components/text_area_with_message_variables.tsx

```diff
@@ -30,8 +30,8 @@ export const TextAreaWithMessageVariables: React.FunctionComponent<Props> = ({
 }) => {
   const [currentTextElement, setCurrentTextElement] = useState<HTMLTextAreaElement | null>(null);
 
-  const onSelectMessageVariable = (variable: string) => {
-    const templatedVar = `{{${variable}}}`;
+  const onSelectMessageVariable = ({ name, useWithTripleBracesInTemplates }: ActionVariable) => {
+    const templatedVar = useWithTripleBracesInTemplates ? `{{{${name}}}}` : `{{${name}}}`;
```

Comment:
Since this is done more than once, it might be nice to have a reusable function (in `action_variables.ts` maybe?)

```
function templateActionVariable(variable: ActionVariable) {
  return variable. useWithTripleBracesInTemplates ? `{{{variable.name}}}` : `{{variable.name}}`;
}
```

Reply:
Good point, fixed in https://github.com/elastic/kibana/pull/84357/commits/33b94347e2d445967e37cfcb1270005c73f26eed 👍